x-pack/plugins/observability_solution/apm/server/routes/services/get_service_agent.ts

```diff
@@ -101,12 +102,16 @@ export async function getServiceAgent({
     return {};
   }
 
-  const { agent, service, cloud } = response.hits.hits[0]._source as ServiceAgent;
+  // TODO fix in the mapper
+  const { agent, service, cloud } = normalizeFields(response.hits.hits[0].fields) as ServiceAgent;
   const serverlessType = getServerlessTypeFromCloudData(cloud?.provider, cloud?.service?.name);
 
   return {
     agentName: agent?.name,
-    runtimeName: service?.runtime?.name,
+    runtimeName:
```

Comment:
@miloszmarcinkowski I think this should be also part of the mapping as there we define the fields so this part can be remove later

Reply:
todo: check if this is still an issue